### DISCUSSION TITLE
File not natively supported

### DISCUSSION THREAD

**Discussion by JuergenZahn** [mifi/lossless-cut#1431] [Q&A] [answered]
Hello,
when I open a File I get the following message: "File not natively supported. Preview may have no audio or low quality. The final export will however be lossless with audio. You may convert it from the menu for a better preview with audio."
I can't find the menu item "convert". What do I have to do?
The file is an unencrypted mpeg2 ts file. I use my Dreambox dm900 (enigma2) for recording.

Yes, that's really a stupid question but I really don't know what to do.

{
  filename: 'XXXXX',
  nb_streams: 7,
  nb_programs: 12,
  format_name: 'mpegts',
  format_long_name: 'MPEG-TS (MPEG-2 Transport Stream)',
  start_time: '68787.577056',
  duration: '7439.399966',
  size: '3917041664',
  bit_rate: '4212212',
  probe_score: 50,
}

**Comment by mifi** [comment, marked as answer]
File->convert to supported format. See also troubleshooting/faq

**Comment by JuergenZahn** [comment]
Hello mifi. Thanks a lot for your fast answer.
I swear. The menu item was not there before (kidding!)
